src/common/SurgePatch.cpp

```diff
@@ -2923,7 +2930,7 @@ unsigned int SurgePatch::save_xml(void **data) // allocates mem, must be freed b
                             // below
                             TiXmlElement mr("modrouting");
                             mr.SetAttribute("source", r->at(b).source_id);
-                            mr.SetAttribute("depth", float_to_str(r->at(b).depth, tempstr));
```

Comment:
So these are not equivalent. float_to_str sets the locale to C to make sure this streams properly if you are in france or whatever with a , as a decimal separator, and std::to_string does not. If we merge this we will break streaming!

There is an STL equivalent but I'll need to figure it out.

Reply:
Modified with updated method float_to_clocalestr() that returns std::string.